crates/store/src/db/migrations/2025062000000_setup/up.sql

```diff
@@ -21,31 +21,31 @@ CREATE TABLE accounts (
 CREATE INDEX idx_accounts_network_prefix ON accounts(network_account_id_prefix) WHERE network_account_id_prefix IS NOT NULL;
 
 CREATE TABLE notes (
-    block_num       INTEGER NOT NULL,
-    batch_index     INTEGER NOT NULL, -- Index of batch in block, starting from 0
-    note_index      INTEGER NOT NULL, -- Index of note in batch, starting from 0
-    note_id         BLOB    NOT NULL,
-    note_type       INTEGER NOT NULL, -- 1-Public (0b01), 2-Private (0b10), 3-Encrypted (0b11)
-    sender          BLOB    NOT NULL,
-    tag             INTEGER NOT NULL,
-    execution_mode  INTEGER NOT NULL, -- 0-Network, 1-Local
-    aux             INTEGER NOT NULL,
-    execution_hint  INTEGER NOT NULL,
-    inclusion_path  BLOB NOT NULL,    -- Serialized sparse Merkle path of the note in the block's note tree
-    consumed        INTEGER NOT NULL, -- boolean
-    nullifier       BLOB,             -- Only known for public notes, null for private notes
-    assets          BLOB,
-    inputs          BLOB,
-    script_root     BLOB,
-    serial_num      BLOB,
+    block_num                INTEGER NOT NULL,
+    batch_index              INTEGER NOT NULL, -- Index of batch in block, starting from 0
+    note_index               INTEGER NOT NULL, -- Index of note in batch, starting from 0
+    note_id                  BLOB    NOT NULL,
+    note_type                INTEGER NOT NULL, -- 1-Public (0b01), 2-Private (0b10), 3-Encrypted (0b11)
+    sender                   BLOB    NOT NULL,
+    tag                      INTEGER NOT NULL,
+    execution_mode           INTEGER NOT NULL, -- 0-Network, 1-Local
+    aux                      INTEGER NOT NULL,
+    execution_hint           INTEGER NOT NULL,
+    inclusion_path           BLOB NOT NULL,    -- Serialized sparse Merkle path of the note in the block's note tree
+    consumed_block_num       INTEGER,
```

Comment:
nit: `consumed_in(_block_num)?` might be easier to understand

Reply:
Happy to change. Interested in what others think about it.

Reply:
I actually don't mind `consumed_at` and `created_at` - though, maybe I'd change `created_at` to `committed_at` since this would be the block in which the note is committed into the chain (but not a strong preference).

Reply:
Will merge committed_at and consumed_at in this PR